packages/react-components/react-motion-components-preview/library/src/components/Scale/ScaleAtoms.ts

```diff
@@ -0,0 +1,36 @@
+import { AtomMotion } from '@fluentui/react-motion';
+
+export const opacityAtom = ({
+  fromOpacity = 0,
+  toOpacity = 1,
+  duration,
+  easing,
+}: {
+  fromOpacity?: number;
+  toOpacity?: number;
+  duration: number;
+  easing: string;
+}): AtomMotion => ({
+  keyframes: [{ opacity: fromOpacity }, { opacity: toOpacity }],
+  duration,
+  easing,
+});
+
+export const scaleAtom = ({
+  fromScale = 0.9,
+  toScale = 1,
+  duration,
+  easing,
+}: {
+  fromScale?: number;
+  toScale?: number;
+  duration: number;
+  easing: string;
+}): AtomMotion => ({
+  keyframes: [
+    { transform: `scale3d(${fromScale}, ${fromScale}, 1)`, visibility: 'visible' },
```

Comment:
The previous implementation used both `visibility` of `hidden` and `visible` between `enter` and `exit` transitions. I haven't tested this new code yet, but I wonder if it functions as expected.